app/javascript/components/report-editor/basic-report-info.jsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { Form, Field } from 'react-final-form';
+import { FinalFormField } from '@manageiq/react-ui-components/dist/forms';
+import { cleanVirtualDom } from '../../miq-component/helpers';
+import { required } from '../../forms/validators';
+
+const BasicReportInfo = ({ name, title, url }) => {
+  // set Initial values if editing
+  const [initialValues] = useState({ name, title });
```

Comment:
this means the PR depends on React 16.8+

what's the timeline for this PR please?

(Cc @mturley - I think we need to finish removing all those `UNSAFE_*` functions from v2v before we can use the one with hooks, right?)

Reply:
> this means the PR depends on React 16.8+

Oh rights 16.8 removed deprecated methods from react. I am using .8 in one of my PRs to. I could help with migration to 16.8 in v2v if need be. I would love to start using 16.8 ASAP as it brings a lot of feature that will improve our codebase.

Reply:
@himdel according to https://reactjs.org/docs/react-component.html, the original method names for those deprecated methods will continue to work until React 17. So this upgrade shouldn't be an issue. But yes, we should probably just run the [`rename-unsafe-lifecycles`](https://github.com/reactjs/react-codemod#rename-unsafe-lifecycles) codemod in v2v. Eventually we should move away from those methods entirely. I just opened https://github.com/ManageIQ/manageiq-v2v/issues/890 to track that in v2v.

Reply:
Thanks! :)